config-overrides.js

```diff
@@ -0,0 +1,30 @@
+const {
+  override,
+  disableChunk,
+  useEslintRc,
+  addWebpackPlugin,
+} = require('customize-cra');
+const path = require('path');
+const RemovePlugin = require('remove-files-webpack-plugin');
+
+const country = process.env.REACT_APP_COUNTRY;
```

Comment:
Should we take the lowercase version?
Also, you need to have a default country, or actually, do not filter the data folder if no country is specified via this environment variable

Reply:
I type it in description, it's already implemented on 15th line, if environment variable not provided RemovePlugin not added

Reply:
perfect, thanks